src/main/java/org/deepsymmetry/beatlink/VirtualRekordbox.java

```diff
@@ -371,9 +371,8 @@ private DeviceUpdate buildUpdate(DatagramPacket packet) {
                         final int player = Util.translateOpusPlayerNumbers(data[0x21]);
                         playerSongStructures.put(player, pssiFromOpus);
                         // Also record the conceptual source slot that represents the USB slot from which this track seems to have been loaded
-                        // TODO we need to check that the track was loaded from a player, and not rekordbox, as well, before trying to do this!
```

Comment:
Rekordbox mode is fully broken. For now we need to fully not support it, until we can put time into getting it right.

Reply:
It seems like it ought to be very easy to support, we can just fall back to the normal mechanisms Beat Link uses because recordbox is happy to respond to metadata requests.